app/controllers/pages_controller.rb

```diff
@@ -33,16 +33,22 @@ def i_belong
     when :enrolled
       champion_path = '/i-belong-champions-pack'
       posters_link_title = 'Request your posters'
-      cta_link_path = i_belong_certificate_path
+      posters_link = 'https://forms.office.com/e/x1FMMzjxhg'
```

Comment:
Another tiny nit: we are causing a trap for our future selves by hard-coding this here, because you already had to put the same link in the 'Request a poster activity'. Should we put it in a translation file and bring it in to both places from there? Or bring it in from the seeded activity in the DB? Tricky - perhaps leave it for now.
```ruby
# db/seeds/activities/
  activity.description = '<a href="https://forms.office.com/e/x1FMMzjxhg">Request</a> and display your set of ‘I Belong in Computer Science’ posters. Consider using with the <a href="https://isaaccomputerscience.org/pages/computer_science_journeys_gallery?examBoard=all&stage=all">interview series</a> to stimulate discussion about computer science related pathways.'
```

Reply:
Tough one; as its the only activity which would have that shared link, it feels a little bit excessive to start putting measures for ALL activities. I think it might be best to let it sit and see if this becomes a pattern.